### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by ilyazzz**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM golang:latest
```
Comment (anatol) [inline Dockerfile:1]: Does the final image include go tools/compilers as well?

I wonder if this docker image can be reduced with the multistage build like described here https://medium.com/swlh/reducing-container-image-size-esp-for-go-applications-db7658e9063a

In fact pacoloco most likely will work with `scratch` images if you go this route.

**Review by anatol — COMMENTED** [review]
Thank you for the contribution.

Could you please add more information to the commit description. What is the docker for? In what case it will be useful? How to deploy it.

And add this information to README as well.

File: Dockerfile (hunk 1 above)
Comment (ilyazzz) [reply Dockerfile]: This was an oversight on my part, I've changed it to use the scratch base for the resulting image. The resulting image is only 5 megabytes now.

**Comment by ilyazzz** [comment]
>Could you please add more information to the commit description. What is the docker for? In what case it will be useful?

There are two main reasons why one would want to run pacoloco in a docker container:
- Running on non-arch systems: my Raspberry Pi runs ubuntu, so if I want to run pacoloco on it i need to build it from source. Docker makes this a lot more convenient.
- Logical separation: example use case would be: I personally use [Traefik](https://github.com/traefik/traefik/) to proxy to all of the services I host so that I don't have to remember on which port something is hosted. Traefik automatically creates routes for docker containers, making the whole setup much less convoluted and easier to manage. I don't need to install any packages, start any services, create proxy rules, all I need is a single docker-compose file and pacoloco is already accessible on a hostname.

These are just my personal use cases, there are many other reasons why one would want to use docker (running pacman cache on a Synology NAS?).

>How to deploy it.

I've added instructions to the README. Though for them to work, pacoloco needs to be pushed to an image registry. I can put it on the docker hub if you're okay with that.

**Comment by anatol** [comment]
thank you, merged!

**Pull request merged**